docs/app/Views/Usage.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react'
+import pkg from 'package.json'
+import {
+  Container,
+  Header,
+  Segment,
+} from 'src'
+import Logo from '../Components/Logo/Logo'
+import { semanticUIDocsURL, semanticUIRepoURL, semanticUICSSRepoURL } from 'docs/app/utils'
+
+const Usage = () => (
+  <Container id='usage-page'>
+    <Segment basic textAlign='center'>
+      <Logo centered size='small' />
+      <Header as='h1' textAlign='center'>
+        Semantic-UI-React
+        <Header.Subheader>
+          {pkg.description}
+        </Header.Subheader>
+      </Header>
+    </Segment>
+
+    <Segment basic padded>
+      <Header as='h2' dividing>Usage</Header>
+      <p>
+        The Semantic UI React package can be installed via NPM:
+      </p>
+      <Segment>
+        <pre>$ npm install {pkg.name}</pre>
+      </Segment>
+      <p>
+        Installing Semantic UI React provides the JavaScript for your components.
+        You'll also need to include a stylesheet to provide the styling for your components.
+        This is the typical pattern for component frameworks, such as Semantic UI or Bootstrap.
+      </p>
+      <p>
+        The method you choose to include the stylesheet in your project will depend on the level
+        of customisation you require.
+      </p>
+    </Segment>
+
+    {/* ----------------------------------------
+     *  Content Delivery Network (CDN)
+     * -------------------------------------- */}
+    <Segment basic padded>
+      <Header as='h2' dividing>Content Delivery Network (CDN)</Header>
+      <p>
+        You can use the default Semantic UI stylesheet by including a Semantic UI CDN link in your
+        <em>index.html</em> file.
+      </p>
+      <p>
+        This is the quickest way to get started with Semantic UI React. You won't be able to use
+        custom themes with this method.
+      </p>
+      <Segment>
+        <pre>
+          {'<link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.2.4/semantic.min.css"></link>'}
+        </pre>
+      </Segment>
+    </Segment>
+
+    {/* ----------------------------------------
+     *  Semantic-UI-CSS package
+     * -------------------------------------- */}
+    <Segment basic padded>
+      <Header as='h2' dividing>Semantic UI CSS package</Header>
```

Comment:
This is great.  Can we update the headers to have some nesting?  Here's an example where I've made `JavaScript` and `CSS` as `h2`s and the three CSS options as `h3` and non-dividing:

![image](https://cloud.githubusercontent.com/assets/5067638/19824031/1f3639ca-9d21-11e6-8603-5b6a52a1287d.png)